near-sdk/src/collections/unordered_set.rs

```diff
@@ -9,7 +9,7 @@ const ERR_INCONSISTENT_STATE: &[u8] = b"The collection is an inconsistent state.
 const ERR_ELEMENT_SERIALIZATION: &[u8] = b"Cannot serialize element with Borsh";
 
 /// An iterable implementation of a set that stores its content directly on the trie.
-#[derive(BorshSerialize, BorshDeserialize)]
+#[derive(BorshSerialize, BorshDeserialize, Debug)]
```

Comment:
This debug impl is broken because `Vector` doesn't have the same conditional implementation. The case where it is different is when `expensive-debug` feature is set and `T` is not constrained by `BorshDeserialize`

Reply:
Another good catch. I know my IDE sometimes will suggest things and if you select an option, do modifications for you. My guess is this is what happened to Chad. https://github.com/near/near-sdk-rs/blame/439126d603dc04effd82d53606a24fac68fc95aa/near-sdk/src/collections/unordered_set.rs#L12
I've removed this and tests are passing and all is well.